src/app.jsx

```diff
@@ -23,18 +23,21 @@ const App = () => {
   return (
     <div className="app">
       <button
+        type="button"
```

Comment:
:open_mouth: I've learned a new rule!
https://developer.mozilla.org/fr/docs/Web/HTML/Element/Button

Reply:
Thank you airbnb :)